charts/kafka/values.yaml

```diff
@@ -42,8 +42,11 @@ resources: {}
   #   memory: 1024Mi
 kafkaHeapOptions: "-Xmx1G -Xms1G"
 
-## Optional Container Security context
-securityContext: {}
+## Kafka pod Security Context
+securityContext:
+  enabled: true
+  fsGroup: 0
+  runAsUser: 0
```

Comment:
I'm happy with this, the confluentinc kafka image requires root, and we can't change that.
If this had to be changed (to be in-line with redis, zookeeper and using runAsUser 1001 etc...), we should switch to the bitnami image.

Reply:
Might be a consideration for the future if we don't want to run as root.